torchbenchmark/util/model.py

```diff
@@ -368,4 +370,24 @@ def inputs_convert(example_inputs):
         if hasattr(self, 'example_inputs'):
             self.example_inputs = inputs_convert(self.example_inputs)
         else:
-            warnings.warn(UserWarning(f"{model_name} example inputs doesn't convert to `channels_last`!"))
+            warnings.warn(UserWarning(f"{model_name} example inputs doesn't convert to `channels_last`!"))       
+
+    def enable_fx_int8(self, quant_engine:str='x86'):
+        torch.backends.quantized.engine = quant_engine
+        try:
+            model, _ = self.get_module()
+            # Get sub modules
+            model, sub_module_list = get_sub_module(model, dict(model.named_modules()), '')
+            if not len(sub_module_list):
+                warnings.warn(UserWarning(f"{self.name} doesn't have submodule can ben quantized!"))
+            model = prepare_sub_module(sub_module_list, model, '', quant_engine)
+            self.set_module(model)
+            # Calibration
```

Comment:
Curious what does Calibration do?

Reply:
With `eager` mode, we may don't need the calibration part. But for `jit` mode, we have the correctness check also, the calibration phase can help most models pass the correctness check with `use_cosine_similarity`. Yes, I'm also noticed that few models don't implemented the `eval()` function, but the `PTQ` quantization is just can work with `eval` test, so decided to add it for all models, if the model not implement the eval function for now, it will be failed. Does it make sense to you?